webapp/public/locale/en/strings.json

```diff
@@ -314,6 +314,7 @@
     "linked_bank_account_successful": "Your bank account has been successfully linked to Suma. Within the next day or two, one of our Member Experience specialists will be in touch to confirm the account.\n\nOnce the account is confirmed, you'll be able to add funds.",
     "no_bank_accounts_warning": "You don't have any bank accounts linked.",
     "no_cards_warning": "You don't have any credit or debit cards linked.",
+    "no_ledgers_warning": "You don't have any payment ledgers.",
```

Comment:
This is not a useful warning or anything anyone can do anything about. Show the generic error message- something went wrong on suma's side and the user doesn't really care what it is.

Reply:
Okay let me know if this works better:
![Screenshot 2024-05-06 at 1 02 17 PM](https://github.com/lithictech/suma/assets/66847768/1fe582d7-bbc1-44dc-8017-8c72ce729b22)